src/specs/NativeStatusBarManagerCompat.ts

```diff
@@ -2,7 +2,7 @@ import type { TurboModule } from 'react-native';
 import { TurboModuleRegistry } from 'react-native';
 
 export interface Spec extends TurboModule {
-  readonly getConstants: () => {};
+  readonly getConstants: () => object;
```

Comment:
@IvanIhnatsiuk can you revert to previous variant? ()=> object is not recognized by codegen and it'll break fabric builds.

Reply:
👀